ccd-definition/AuthorisationCaseField/CareSupervision/court-admin.json

```diff
@@ -2377,7 +2377,7 @@
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "CaseFieldID": "courtBundleList",
     "UserRole": "caseworker-publiclaw-courtadmin",
-    "CRUD": "R"
+    "CRUD": "CRU"
```

Comment:
why are we updating the admin permissions to be more than just read? the ticket doesn't mention them adding or updating the list

Reply:
if it it for the auth complex type then it might be worth just having `R` and `U` permissions and add a comment to it stating that the `U` is to enable functionality in the auth complex type. (same for the other comments related to this one)

Reply:
I added a new generated complex type field called 'courtBundleNC'. This field is supposed to be shown to the non-LA and non-HMCTS users only and doesn't contain the confidential documents. The existing 'courtBundle' complex type field will only be shown to LA and HMCTS users. In order to restrict these fields, I will need to mark some roles as 'U' only. If I leave 'courtBundleList' as 'R', it complains about the parent type has lower access level than the child type. 

In order to restrict the access to 'courtBundle' and 'courtBundleNC' properly, the permission on the higher level need to up and hence I change them to 'CRU'. Note that this is done the same way for other document sub-event like 'Respondent statement' etc. 

I can try changing it to 'R' and 'U' only and check it still work.

Reply:
All the comments regarding the 'CRU' permission has been update to 'CU'